src/rgw/rgw_sal_motr.cc

```diff
@@ -3202,10 +3205,16 @@ int MotrMultipartUpload::complete(const DoutPrefixProvider *dpp,
   if (rc < 0) {
     return rc == -ENOENT ? -ERR_NO_SUCH_UPLOAD : rc;
   }
+  rgw::sal::Attrs temp_attrs;
+  // MotrObject::Meta meta;
   rgw_bucket_dir_entry ent;
   bufferlist& blr = bl;
   auto ent_iter = blr.cbegin();
   ent.decode(ent_iter);
+  decode(temp_attrs, ent_iter);
```

Comment:
we may not need anything depending on tests, if we get tags as part of attrs here already